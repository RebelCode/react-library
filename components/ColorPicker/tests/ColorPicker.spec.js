import React from "react";
import { render, cleanup } from "@testing-library/react";
import { ColorPicker } from "../index";

afterEach(cleanup);

describe("ColorPicker", () => {
  it("should take a snapshot", () => {
    const { asFragment } = render(<ColorPicker />);

    expect(asFragment(<ColorPicker />)).toMatchSnapshot();
  });

  it("renders correctly", () => {
    const { queryByLabelText } = render(<ColorPicker />);

    expect(queryByLabelText("ColorPicker")).toBeTruthy();
  });
});
