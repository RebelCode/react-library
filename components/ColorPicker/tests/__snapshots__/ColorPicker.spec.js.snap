// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorPicker should take a snapshot 1`] = `
<DocumentFragment>
  <button
    class="button"
  >
    <span
      aria-label="ColorPicker"
      class="colorPreview"
      data-testid="colorpicker"
      style="background-color: rgb(255, 255, 255);"
    />
  </button>
</DocumentFragment>
`;
